src/Examples/Advanced/CustomUnitModels/methanol_param_VLE.py

```diff
@@ -127,35 +122,37 @@ def build(self):
 
         self.gamma = Param(within=NonNegativeReals, mutable=True, default=gamma, doc='Ratio of Cp to Cv')
 
-        self.Cp = Param(within=NonNegativeReals, mutable=True, default=Cp, doc='Constant pressure heat capacity [MJ/(kgmol K)]')
+        self.Cp = Param(within=NonNegativeReals, mutable=True, default=Cp,
+#                        units=pyunits.MJ/pyunits.kmol/pyunits.K,
+                        doc='Constant pressure heat capacity [MJ/(kmol K)]')
 
     @classmethod
     def define_metadata(cls, obj):
         """Define properties supported and units."""
         obj.add_properties(
-            {'flow_mol': {'method': None, 'units': 'kgmol/s'},
+            {'flow_mol': {'method': None, 'units': 'kmol/s'},
              'mole_frac': {'method': None, 'units': 'no unit'},
              'temperature': {'method': None, 'units': '100K'},
```

Comment:
This is going to be an issue. The current implementation of units does not allow a lot of freedom here, and everything needs to be based on the units defined in the units metadata.

So, using 100*K (centaKelvins?) is possible, it means that any unit of temperature will be in 100*K. Similarly, it is possible to set thing up so that pressure is in MPa, but it means that you have to use mass/length/time units that work out that way and use them everywhere else.

Reply:
@jghouse88 explained that the scaling originated from the source example. Removing the factor of 100 in the enthalpy balance (state block file near the bottom) seemed to make the compressor work error worse so I kept it for now.

Reply:
The problem here is that scaling should be kept separate from units. The fact that the model has solver issues implies that the scaling tools should be applied, rather than that the units should be changed.